go.mod

```diff
@@ -127,6 +127,6 @@ require (
 	k8s.io/component-base v0.18.19
 	k8s.io/cri-api v0.18.19
```

Comment:
These here should all get bumped too - anything that's k8s.io v0.x.y should go to v0.x.y+1. The convention is that the kubernetes submodules are versioned on major less than k8s.io/kubernetes itself.

klog, utils, client-go have their own versioning scheme that's detached from this, but you should be able to tell from the current go.mod which ones need to be updated.

Reply:
Updated the missing v0.x.y modules